src/Add_remove.js

```diff
@@ -0,0 +1,28 @@
+const TodoLoad = (todos) => {
+  todos = JSON.parse(localStorage.getItem('todos')) || [];
+  for (let i = 0; i < todos.length; i += 1) {
+    todos[i].index = i + 1;
+  }
+  return todos;
+};
+
+class Todo {
+  constructor(description, index = 0, completed = false) {
+    this.index = index;
+    this.description = description;
+    this.completed = completed;
+  }
+
+  addTodo = (task, todos) => {
+    todos.push(task);
+    localStorage.setItem('todos', JSON.stringify(todos));
+  }
+
+  deleteTodo = (e, todos) => {
+    e.target.parentElement.remove();
+    todos.splice(e.target.parentElement.id, 1);
+    localStorage.setItem('todos', JSON.stringify(todos));
+  }
+}
```

Comment:
- [ ] When a to-do list item is deleted, please remember to update all remaining items' indexes, so they represent the current list order and are unique 👍